libmscore/tie.cpp

```diff
@@ -287,8 +287,11 @@ void TieSegment::computeBezier(QPointF p6o)
       shapePath.cubicTo(p4 +p4o + th, p3 + p3o + th, QPointF());
 
       // translate back
+      double y = pp1.y();
+      if (staff()->isTabStaff(slurTie()->tick()))
+          y += (_spatium * (slurTie()->up() ? -0.2 : 0.2));
```

Comment:
Could you please add the variable which describes the meaning of `0.2` magic number?

Reply:
OK.  It's not very magical. It's my best guess.  I asked for feedback and got none, so here it is. I'll declare it as a const double.

Reply:
I named it `offsetFactor`.  This is an extra offset for tablature.

Reply:
See here for a similar, existing line of code:
https://github.com/musescore/MuseScore/blob/b3adb50bd655bc943c502af8d6dc22fd9422f81a/libmscore/note.cpp#L1116
That's where I got the idea for this offset.